Fuse/Classes/Fuse.swift

```diff
@@ -348,16 +348,26 @@ extension Fuse {
         
         var items = [SearchResult]()
         
+        // Serialize writes to `items`, for thread safety.
+        // This label is non-unique but that should be fine as we don't expect
+        // to need to debug work items running on this queue.
+        let itemsQueue = DispatchQueue(label: "fuse.items.queue")
```

Comment:
Let me know if you have any thoughts about how to make these labels unique. I kind of wish that we could just omit the label entirely if we don't expect to debug items on the queue. The documentation says that the label is optional, but I think they broke that (if it was ever the case) when porting the API to Swift, because the label argument is a non-optional String.

Reply:
That's a good call. Had the same problem with labeling. Naming looks fine though.